src/core/index.ts

```diff
@@ -75,16 +96,39 @@ interface BellPromptConfig {
     };
     colors?: {
         bell?: {
+            fg?: string;
```

Comment:
Was this just missing from iface?

Reply:
Seems to be showing as addition from refactoring files.

Reply:
Looks like file change malarky there!